src/src/auth/auth.controller.ts

```diff
@@ -23,4 +23,10 @@ export class AuthController {
     public verifyOtp(@Body() req: OtpVerifyDTO, @Res() response: Response) {
         return this.authService.verifyOtp(req, response);
     }
+
+    @Post('/reset-password')
```

Comment:
lets rename route as reset-password-otp
and 
methods as resetPasswordUsingOtp

Reply:
done